src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ]  I cannot run the tests according to your instructions, because I'm greeted by a syntax error:

![image](https://user-images.githubusercontent.com/57421823/203661540-ad91086b-7261-4a37-87f4-50d3e3a8b802.png)

Please, kindly check if this is running correctly on your end.  If it is not, kindly debug your tests so your app works as intended. 💪